src/pilot/assign_role.py

```diff
@@ -1015,9 +1034,17 @@ def get_drives(drac_client):
     spinners = []
     ssds = []
     physical_disks = drac_client.list_physical_disks()
-    for physical_disk in physical_disks:
+    virtual_disks = drac_client.list_virtual_disks()
+    raid_disks = [vd.physical_disks for vd in virtual_disks
+                  if vd.raid_level == '1'][0]
```

Comment:
If for some reason there are multiple RAID 1s, we don't know which one has the OS on it, so we need to exclude all of the physical disks for all of the RAID1s, otherwise Ceph could overwrite the OS when it initializes the OSDs.  So I think the above needs to change to collect all of the physical disks on all the RAID1s.

Reply:
I have modified the above code as per required changes.